pages/linux/iptables.md

```diff
@@ -22,6 +22,10 @@
 
 `sudo iptables -D {{chain}} {{rule_line_number}}`
 
-- Save iptables configuration:
+- Save iptables configuration to file (single table):
```

Comment:
How about - `Save iptables configuration of a given table to a file` ? So that the sentence doesn't stutter.